server/routes/index.js

```diff
@@ -0,0 +1,22 @@
+import Businesses from '../controller/business';
+import Users from '../controller/user';
+import Reviews from '../controller/reviews'; // eslint-disable-line no-unused-vars
+
+ import businesses from '../model/business'; // eslint-disable-line no-unused-vars
```

Comment:
This is awesome. I see you are using es6. You don't need to import things that would not be used. I feel the business models are not needed here.